academy/3-ibc/4-clients.md

```diff
@@ -220,7 +224,7 @@ Assume that the initial ConsensusState was created at block 50, but you want to
 
 To update the `ConsensusState` of the counterparty on the client, an `UpdateClient` message containing a `Header` of the chain to be updated must be submitted by a relayer. For all IBC client types, Tendermint or otherwise, this `Header` contains the information necessary to update the `ConsensusState`. However, IBC does not dictate what the `Header` must contain beyond the basic methods for returning `ClientType` and `GetClientID`. The specifics of what each client expects as important information to perform a `ConsensusState` update will be found in each client implementation.
 
-For example, the Tendermint client `Header` looks like [this](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/tendermint.pb.go):
+For example, the Tendermint client `Header` looks like [this](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/tendermint.pb.go#L198):
```

Comment:
I'd recommend not using a link to main since the code may change again. May use a specific commit or release tag?